docs/en/getting-started.md

```diff
@@ -1 +1,137 @@
 # Getting Started
+
+Get up and running on with the official `vue-test-utils` is straight forward.
```

Comment:
Maybe try "Getting up and running with the official `vue-test-utils` package is straight forward." or "Getting up and running with `vue-test-utils` is straight forward." - there's a couple words missing as it is now